erpnext/accounts/report/sales_register_by_item/sales_register_by_item.py

```diff
@@ -78,27 +76,13 @@ def _execute(filters=None, additional_table_columns=None, additional_query_colum
 			'sales_order': d.sales_order,
 		})
 
-		total_tax = 0
-		for tax in tax_columns:
-			item_tax = itemised_tax.get(d.name, {}).get(tax, {})
-			row.update({
-				frappe.scrub(tax + ' Rate'): item_tax.get("tax_rate", 0),
-				frappe.scrub(tax + ' Amount'): item_tax.get("tax_amount", 0),
-			})
-			total_tax += flt(item_tax.get("tax_amount"))
-
-		row.update({
-			'total_tax': total_tax,
-			'total': d.base_net_amount + total_tax,
-			'currency': company_currency
-		})
 
 		if filters.get('group_by'):
 			row.update({'percent_gt': flt(row['total']/grand_total) * 100})
 			group_by_field, subtotal_display_field = get_group_by_and_display_fields(filters)
 			data, prev_group_by_value = add_total_row(data, filters, prev_group_by_value, d, total_row_map,
-				group_by_field, subtotal_display_field, grand_total, tax_columns)
-			add_sub_total_row(row, total_row_map, d.get(group_by_field, ''), tax_columns)
+				group_by_field, subtotal_display_field, grand_total)
+			add_sub_total_row(row, total_row_map, d.get(group_by_field, ''))
```

Comment:
@harshmule-git did you tested locally. please test and review from start to end each and every calling function.
add_total_row contain tax_column when its defined should be removed from there too